### PR TITLE
Add new icon `nuxt`

### PR DESCRIPTION
**Pull request by castrogarciajs**
Hello Miguel, me again

I realized that [Nuxt](https://nuxt.com/) did not have icons... I hope you like it. 🙈

## Preview

https://github.com/miguelsolorio/vscode-symbols/assets/116913262/5b741378-1912-4290-86d2-6853cf3e3265

### CODE REVIEW

File: src/symbol-icon-theme.json [hunk 1]
```diff
@@ -1280,6 +1285,7 @@
 		".vscode": "folder-blue",
 		".next": "folder-gray",
 		".turbo": "folder-red",
+		".nuxt": "folder-green",
```
Comment (FabianMontoya) [inline src/symbol-icon-theme.json:1288]: Maybe create a folder with the small nuxt icon as variant could be better

Comment (FabianMontoya) [reply src/symbol-icon-theme.json:1288]: Like Github folder for example

Comment (castrogarciajs) [reply src/symbol-icon-theme.json:1288]: I like the idea, I'll keep it in mind.

Comment (castrogarciajs) [reply src/symbol-icon-theme.json:1288]: Look at PR #181

Comment (castrogarciajs) [reply src/symbol-icon-theme.json:1288]: @FabianMontoya I was testing and it really doesn't look very good, the icon looks too small

**Pull request closed without merging**